test/cri-containerd/execcontainer_test.go

```diff
@@ -122,3 +130,19 @@ func Test_ExecContainer_RunAs_LCOW(t *testing.T) {
 		t.Fatalf("failed to start container with runas option: error: %v, exitcode: %d", errorMsg, exitCode)
 	}
 }
+
+func Test_ExecContainer_LCOW_HasEntropy(t *testing.T) {
```

Comment:
Can we switch the order. `Test_ExecContainer_HasEntropy_LCOW`? The goal was to be able to run things like `-run=Test_ExecContainer`, `-run=Test_ExecContainer_HasEntropy`. Maybe there are better ways to do filters like these?